.github/workflows/ci.yml

```
@@ -292,3 +302,50 @@ jobs:
                 tag: '${{ github.sha }}'
               }
             })
+
+  deploy-on-neotoolsuite-dev:
+    if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
+    name: Deploy to Neotoolsuite Dev
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: '0'
+          persist-credentials: false
+
+      - name: Checkout valhalla for values
+        uses: actions/checkout@v4
+        with:
+          ref: dev
+          sparse-checkout: |
+            /cluster/neotoolsuite-dev/neoboard/values-neotoolsuite-dev.yaml
+          sparse-checkout-cone-mode: false
+          repository: nordeck/valhalla
+          token: ${{ secrets.PACKAGE_TOKEN }}
+          path: valhalla
+          persist-credentials: false
+
+      - name: Create Kubeconfig
+        run: |
+          # Extract the base64 encoded config data and write this to the KUBECONFIG
+          echo "${{ secrets.KUBECONFIG_NEOTOOLSUITE_DEV }}" | base64 -d > /tmp/config
+          chmod 600 /tmp/config
+          echo "KUBECONFIG=/tmp/config" >> $GITHUB_ENV
+
+      - name: Deploy k8s to cluster
+        uses: wahyd4/kubectl-helm-action@master
+        env:
+          KUBE_CONFIG_DATA: ${{ secrets.KUBECONFIG_NEOTOOLSUITE_DEV }}
+        with:
+          args: |
+            helm upgrade --install neoboard ./charts/matrix-poll-widget -f ./valhalla/cluster/neotoolsuite-dev/neoboard/values-neotoolsuite-dev.yaml -n matrix --set image.tag="latest"
```

this used the wrong chart
```suggestion
            helm upgrade --install neoboard ./charts/matrix-neoboard-widget -f ./valhalla/cluster/neotoolsuite-dev/neoboard/values-neotoolsuite-dev.yaml -n matrix --set image.tag="latest"
```
but it needs a further change to use the argo tag instead of latest

---

I guess this is it?
```suggestion
            helm upgrade --install neoboard ./charts/matrix-neoboard-widget -f ./valhalla/cluster/neotoolsuite-dev/neoboard/values-neotoolsuite-dev.yaml -n matrix --set image.tag="${{ github.sha }}"
```